modules/tinymce/src/core/main/ts/api/dom/DOMUtils.ts

```diff
@@ -460,10 +447,19 @@ const DOMUtils = (doc: Document, settings: Partial<DOMUtilsSettings> = {}): DOMU
   };
 
   const setStyles = (elm: string | Element | Element[], stylesArg: StyleMap) => {
-    setStyle(elm, stylesArg);
```

Comment:
Changed this to make it work better with the types, so I moved the internals of `setStyle` out to separate functions that are called by both `setStyle` and `setStyles`.